lib/auth/2fa.ts

```diff
@@ -9,7 +9,10 @@ export const generateVerificationCode = async () => generateTokenCode(5);
 
 export const sendVerificationCode = async (email: string, verificationCode: string) => {
   try {
-    const notify = new NotifyClient("https://api.notification.canada.ca", NOTIFY_API_KEY);
+    // Avoids test accounts being blocked by Notify
+    const notifyUrl =
```

Comment:
Just a thought ...

What if we wrap the Notify Client 



So instead of pulling in

```
import { NotifyClient } from "notifications-node-client";
```


We 
import { NotifyClient } from "notify";

```
// notify.ts

const mockNotify = {
   sendEmail: () =>{
    log("email sent ...")
   }
}

export const  NotifyClient = () =>{

  if(process.env.APP_ENV === "test"){
   ... return  mockNotify;
  }
  
  else 
  .. 
  
  return  new NotifyClient("https://api.notification.canada.ca", NOTIFY_API_KEY);
  
  
  
}

const notify = new NotifyClient("https://api.notification.canada.ca", NOTIFY_API_KEY);
```

Reply:
Looks good IMO. That would remove all the if-statements and keep our code a bit cleaner. We'd just need to remember to use the mock in tests

Reply:
>  mock in tests

If we update the import to use out wrapper it should figure itself out based in the env

Reply:
Oh I see, notify.js would be used in place globally, got it. It could also be a convenient place to add any notify specific customization in the future